notebooks/default-prep-script.ipynb

```diff
@@ -0,0 +1,417 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e3736f91-5695-4beb-8591-227eb444adc0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "import datetime\n",
+    "import math\n",
+    "import numpy\n",
+    "import os\n",
+    "import requests\n",
+    "import stat\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import sycamore\n",
+    "from sycamore.functions import HuggingFaceTokenizer, TextOverlapChunker\n",
+    "from sycamore.llms import OpenAI, OpenAIModels\n",
+    "from sycamore.transforms.embed import SentenceTransformerEmbedder\n",
+    "from sycamore.transforms.extract_entity import OpenAIEntityExtractor\n",
+    "from sycamore.transforms.merge_elements import GreedyTextElementMerger\n",
+    "from sycamore.transforms.partition import UnstructuredPdfPartitioner, HtmlPartitioner\n",
+    "\n",
+    "index = \"demoindex0\""
+   ]
+  },
+  {
```

Comment:
I think this cell is obsolete now also, it's only used to print out file paths not in the execution.